docs/components/modeler/desktop-modeler/element-templates/defining-templates.md

```diff
@@ -331,6 +348,26 @@ Configures the [task type](../../../bpmn/service-tasks/#task-definition) for a s
 
 The `zeebe:property` binding allows you to set any arbitrary property for an outside system. It does not impact execution of the Zeebe engine.
 
+#### `bpmn:Message#property`
+
+| **Binding `type`**          | `bpmn:Message#property`                            |
+| --------------------------- | -------------------------------------------------- |
+| **Valid property `type`'s** | `String`<br />`Text`<br />`Hidden`<br />`Dropdown` |
+| **Binding parameters**      | `name`: The name of the property                   |
+| **Mapping result**          | `<bpmn:Message [name]="[userInput]" />`            |
```

Comment:
```suggestion
| **Mapping result**          | `<bpmn:message [name]="[userInput]" />`            |
```

Reply:
We should be consistent here. XML tag names are lowercase, right?

Reply:
Correct.